src/lib/types/experimental.ts

```diff
@@ -260,10 +259,6 @@ const flags: IFlags = {
         process.env.EXPERIMENTAL_EDGE_OBSERVABILITY,
         false,
     ),
-    registerFrontendClient: parseEnvVarBoolean(
-        process.env.UNLEASH_EXPERIMENTAL_REGISTER_FRONTEND_CLIENT,
-        false,
-    ),
```

Comment:
I know in some cases we might wanna keep this or at least keep the option of disabling this with an env var... probably not in this particular case though...

Reply:
Do we have some examples of that so I can have a look?

Maybe a follow up to our action could be to listen to PR comments, so it can adjust the cleanup. That way we can ask it to keep the conditions and the env var 🤔 Otherwise I'm not sure this will have a stable pattern it can follow.

Reply:
Yesterday I finished 100% rollout of this feature. You can mark it as completed in UI, create another PR and merge if you want to :) This way ful e2e flow can be tested with this flag

Reply:
That's good timing @kwasniew! Maybe you can help us test the flow e2e with this flag and give us your feedback.

Technically you can try it right away, but give me some time to polish a few things and I'll let you know when it's a good time to try it, if that's okay.